package.json

```diff
@@ -122,7 +122,8 @@
     },
     "express": {
       "path-to-regexp": "0.1.10"
-    }
+    },
```

Comment:
Why do we need this?

Reply:
If not indicated express is using a path-to-regexp version that causes the mibi server to crash during startup.